starport/templates/ibc/oracle/dynamic/x/{{moduleName}}/types/{{queryName}}_test.go.plush

```diff
@@ -0,0 +1,39 @@
+package types
+
+import (
+	"testing"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMsg<%= queryName.UpperCamel %>Data_ValidateBasic(t *testing.T) {
+	tests := []struct {
+		name string
+		msg  Msg<%= queryName.UpperCamel %>Data
+		err  error
+	}{
+		{
+			name: "invalid address",
+			msg: Msg<%= queryName.UpperCamel %>Data{
+				Creator: "invalid_address",
+			},
+			err: sdkerrors.ErrInvalidAddress,
+		}, {
+			name: "valid address",
+			msg: Msg<%= queryName.UpperCamel %>Data{
+				Creator: "cosmos1uzv4v9g9xln2qx2vtqhz99yxum33calja5vruz",
```

Comment:
The address can use another prefix than `cosmos`

We should be able to generate an address with the correct prefix with this oneliner:

```
import "sdk "github.com/cosmos/cosmos-sdk/types""
import "github.com/cosmos/cosmos-sdk/crypto/keys/ed25519"

sdk.AccAddress(ed25519.GenPrivKey().PubKey().Address()).String()
```